be/src/olap/rowset/segment_v2/inverted_index_reader.cpp

```diff
@@ -17,30 +17,288 @@
 
 #include "olap/rowset/segment_v2/inverted_index_reader.h"
 
-#include "common/status.h"
+#include <CLucene/search/BooleanQuery.h>
+#include <CLucene/search/PhraseQuery.h>
+
+#include "common/config.h"
+#include "gutil/strings/strip.h"
+#include "io/fs/file_system.h"
+#include "olap/key_coder.h"
+#include "olap/rowset/segment_v2/inverted_index_compound_directory.h"
+#include "olap/rowset/segment_v2/inverted_index_compound_reader.h"
+#include "olap/rowset/segment_v2/inverted_index_desc.h"
+#include "olap/tablet_schema.h"
+#include "olap/utils.h"
+#include "runtime/string_value.h"
+#include "util/time.h"
 
 namespace doris {
 namespace segment_v2 {
 
+bool InvertedIndexReader::_is_match_query(InvertedIndexQueryType query_type) {
+    return (query_type == InvertedIndexQueryType::MATCH_ANY_QUERY ||
+            query_type == InvertedIndexQueryType::MATCH_ALL_QUERY ||
+            query_type == InvertedIndexQueryType::MATCH_PHRASE_QUERY);
+}
+
 bool InvertedIndexReader::indexExists(io::Path& index_file_path) {
     bool exists = false;
     RETURN_IF_ERROR(_fs->exists(index_file_path, &exists));
     return exists;
 }
 
+std::vector<std::string> FullTextIndexReader::get_analyse_result(
+        const std::wstring& field_name, const std::wstring& value,
+        InvertedIndexQueryType query_type, InvertedIndexParserType analyser_type) {
+    std::vector<std::string> analyse_result;
+    std::shared_ptr<lucene::analysis::Analyzer> analyzer;
+    if (analyser_type == InvertedIndexParserType::PARSER_STANDARD) {
```

Comment:
should keep consistent with writer analyzer

Reply:
here reader default analyzer is SimpleAnalyzer, writer default analyzer is also SimpleAnalyzer